cpk-core/ivy.xml

```diff
@@ -48,9 +48,14 @@
     <dependency org="com.sun.jersey" name="jersey-client" rev="1.16" conf="test->default" transitive="false"/>
 
 
+
     <!--  third-party dependencies for tests -->
     <dependency org="com.google.guava" name="guava" rev="17.0" conf="test->default"/>
-     <dependency org="pentaho"             name="metastore"           rev="${dependency.pentaho-metastore.revision}"  conf="test->default" changing="true" />
+    <dependency org="pentaho"             name="metastore"           rev="${dependency.pentaho-metastore.revision}"  conf="test->default" changing="true" />
+    <dependency org="pentaho-kettle" name="kettle-json-plugin" rev="${dependency.kettle.revision}" transitive="false" conf="test->default"  changing="true">
```

Comment:
There is a concrete version property for the kettle-json-plugin => kettle-json-plugin.version. Please update the version property name.
http://devci.pentaho.com/job/7.1-snapshot/view/control/job/build-resources/ws/7.1-SNAPSHOT/version.properties/*view*/